content/en/v3/develop/reference/jx/gitops/git/setup/_index.md

```diff
@@ -25,14 +25,50 @@ This is typically used in a pipeline to ensure git can do commits.
   jx-gitops git setup
 
   ```
+  
+### Details
+
+The `jx gitops git setup` command ensures that we can authenticate with configured Git server by configuring the local
+credentials file in the home directory. This command tries to ensure following things:
+
+- The user can be authenticated with Git provider (for example Github)
+- An email is associated with each automated commit message
+
+These credentials are written to `${HOME}/git/credentials` file, where the `${HOME}` directory is determined as:
+
+- value stored in `XGD_CONFIG_HOME` environment variable or
+- value stored in `HOME` environment variable or
+- value stored in `USERPROFILE` environment variable or
+- as current directory `.`
+
+The credentials are determined by reading out the `jx-requirements.yaml` from the cluster repository and `jx-boot`
+Secret resource provisioned together with `jx-git-operator` in your Kubernetes namespace.
+
+The Git username and email are preferentially determined from `PipelineUser` field from `jx-requirements.yaml`, but if
+they are not available there then default email address `jenkins-x@googlegroups.com` is used (which is the reason why
+you are seeing commits from `pow-devops2020` user made on your repositories). If the username could not be determined
```

Comment:
Shouldn't be the case anymore for new users, not sure if pow-devops2020 should be mentioned.

Reply:
Does this mean that in newer Jx versions email is going to be mandatory part of git access configuration? If yes then I agree.

Reply:
I don't know, but it would make sense IMO.

Reply:
@jstrachan has commented on this topic in Slack conversation, that something was in the making about usage of `pow-devops2020` user. Maybe he can provide more info?